README.md

```diff
@@ -487,6 +487,16 @@ end
 
 ```
 
+### Development
+
+You can get development environment. It requires [vagrant](https://www.vagrantup.com/).
```

Comment:
Yes, please remove it. Rails core development uses vagrant for development because there are many dependencies that are tested against (multiple db versions and such). All this gem requires is ruby 2.2+ and ActiveSupport 3.2+. If someone cannot set that up, then they can set up vagrant or some other vm themselves and shouldn't need this codebase to do it for them.